### DISCUSSION TITLE
1.4.0-beta0

### DISCUSSION THREAD

**Discussion by jglev** [jglev/obsidian-carry-forward#14] [Announcements]
- Added settings toggle for `Copied` Notice (#12)
- Added support for multiple cursors (#11) 

@FelipeRearden, would you be willing to see whether these work for you, too? : )

<hr /><em>This discussion was created from the release <a href='https://github.com/jglev/obsidian-carry-forward/releases/tag/1.4.0-beta0'>1.4.0-beta0</a>.</em>

**Comment by FelipeRearden** [comment]
Hello @jglev !!!!!!!!!!!!

## Version 1.4.0-beta0

#12 -> working perfect on version 1.4.0-beta0 Obsidian 0.13.23

#11 -> See Below

### Test Enviroment

Obsidian 0.13.23
Real Note with 14 highlights (real workflow)
- one highlight per line
- More than one highlight per line
Fake Note with 4 highlights
- one highlight per line
- More than one highlight per line

All 12 Commands where tested

Disable Notification toggle ON

### Source
One Single Cursor
Multiple Cursors 

### Console Error 
Zero

### Carry + Apple Patterns (clipboard line by line)
Working perfect

### Result 
From the 12 commands applied the two sources (only single cursor and multiple cursors), 4 commands didn't work 
- Copy Link to Line (Link Text From Clipboard)
- Copy Embed Link to Line (Link Text From Clipboard)
- Copy Selection with each line link to its copied source (Link Text From Clipboard)
- Copy Selection with first line link to its copied source (Link Text From Clipboard)

### Bug -> Link Text From Clipboard

The commands that uses the clipboard to replace the content of the `{{LinkText}}` are not working

It happens with SINGLE CURSOR
It happens with MUTIPLR CURSORS

Example: 
- select the text in the clipboard is `Apple`

after hitting the Carry-Forward Command:
- its does not create the block ref id
- When hitting CRTL/CMD+V to paste the clipboard is empty 

---

Sorry for bring problems :)
Let me know if you need anymore tests or any other information:)


Have a great day and thank you very much !!!!!

**Comment by jglev** [comment]
@FelipeRearden, gah, thank you! I completely missed those bugs!

I _think_ I've addressed them (hopefully without adding additional issues) in [`beta1`](https://github.com/jglev/obsidian-carry-forward/releases/tag/1.4.0-beta1) just now.

**Reply by FelipeRearden** [reply]
> @FelipeRearden, gah, thank you! I completely missed those bugs!
> 
> 
> 
> I _think_ I've addressed them (hopefully without adding additional issues) in [`beta1`](https://github.com/jglev/obsidian-carry-forward/releases/tag/1.4.0-beta1) just now.

No problem at all @jglev :) 

Thanks for the quick fix!!!! I'm doing the tests now and I get to you in 30 minutes.

**Comment by FelipeRearden** [comment]
## Version 1.4.0-beta1

### Test Enviroment

Obsidian 0.13.23
Real Note with 14 highlights (real workflow)
- one highlight per line
- More than one highlight per line
Fake Note with 4 highlights
- one highlight per line
- More than one highlight per line

All 12 Commands where tested

Disable Notification toggle ON

### Source
One Single Cursor
Multiple Cursors 

### Console Error 
Zero

### Carry + Apple Patterns (clipboard line by line)
Working perfect

### Result 
From the 12 commands applied the two sources (only single cursor and multiple cursors) everything is working perfect !!!!!!!!!

In my humble opinion, we are ready for the merge !!!!!!

Thank very much @jglev !!!! Once again, you make my Obsidian dreams a reality. I'm very grateful to you !!!!!!!!

**Reply by jglev** [reply]
@FelipeRearden, thank you *so much!* I'm very grateful to you for your help and your time! :100: This release is better because of your help.

I've released [`1.4.0`](https://github.com/jglev/obsidian-carry-forward/releases/tag/1.4.0) just now. :rocket:

**Reply by FelipeRearden** [reply]
> @FelipeRearden, thank you *so much!* I'm very grateful to you for your help and your time! :100: This release is better because of your help.
> 
> 
> 
> I've released [`1.4.0`](https://github.com/jglev/obsidian-carry-forward/releases/tag/1.4.0) just now. :rocket: 

Thank you very much @jglev for the kind words !!!!!!!

It's always a pleasure to work with you in a new release !!!!

I wish you a fantastic week !!!!!
